go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/golang-jwt/jwt v3.2.2+incompatible
```

Comment:
Need to update gomod to 1.19 🔝

Reply:
Didn't upgrade the package as it is require migration of our JWT format.
Keeping it and will address it in a different pr if needed.